allennlp/commands/evaluate.py

```diff
@@ -0,0 +1,91 @@
+from typing import Dict, Any
+import argparse
+import json
+import os
+import logging
+
+from allennlp.common.params import Params, replace_none
+from allennlp.data import Vocabulary, Dataset
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.iterators import DataIterator
+from allennlp.models.model import Model
+from allennlp.nn.util import arrays_to_variables, device_mapping
+
+import torch
+import tqdm
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+def add_subparser(parser: argparse._SubParsersAction) -> argparse.ArgumentParser:  # pylint: disable=protected-access
+    description = '''Evaluate the specified model + dataset'''
+    subparser = parser.add_parser(
+            'evaluate', description=description, help='Evaluate the specified model + dataset')
+    subparser.add_argument('--config_file',
+                           type=str,
+                           required=True,
+                           help='path to the configuration file that trained the model')
+    subparser.add_argument('--weights_file',
```

Comment:
my thinking was that the path was recoverable, but you might want to evaluate using different checkpoint weights or best weights? I guess I could default to serialization_prefix/best.th?